package.json

```diff
@@ -1,16 +1,14 @@
 {
   "name": "siren-parser",
   "version": "9.0.0",
-  "main": "dist/index.js",
-  "module": "src/index.js",
```

Comment:
I don't actually see this `module` in the docs for package.json, so I'm thinking it might have been an old thing that's been removed? Either way, not needed anymore.

Reply:
I think it was how you could expose a different entrypoint for `import` vs `require` -- not needed though since this is now `type: module`.